src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -3305,6 +3310,26 @@ public function is_development_environment() {
 		return in_array( wp_get_environment_type(), array( 'development', 'local' ), true );
 	}
 
+	/**
+	 * Returns the URL used for checking page caching. By default, this is the home URL.
+	 *
+	 * @since 6.8.0
+	 *
+	 * @return string The URL for checking page caching.
+	 */
+	public function get_persistent_page_cache_url() {
+		/**
+		 * Filters the URL used for checking page caching in the Site Health tool.
+		 *
+		 * This filter allows you to set a custom URL to determine if persistent page caching is enabled.
+		 *
+		 * @since 6.8.0
+		 *
+		 * @param string $url The URL used for checking page caching. Default home URL.
+		 */
+		return apply_filters( 'site_status_persistent_page_cache_url', home_url( '/' ) );
```

Comment:
The original ticket mentions desire to test multiple different sites, for example on a multilingual or ecommerce site. But this is not possible if this is just a single filterable URL.

If testing of multiple pages is desired, then this should be an array of URLs.

Reply:
+1